### ISSUE TITLE
Issue with bundling

### ISSUE THREAD

**Issue by kinsomicrote** [christopherstyles/ngp_van#32]
Hello @christopherstyles 

I am faced with this issue when I run `bundle install`

```
The gem ngp_van-0.8.0 can't be installed because the security policy didn't allow it, with the message: certificate /CN=christopherstyles/DC=gmail/DC=com not valid after 2019-05-02 23:27:30 UTC
```

Any idea how to resolve this?

Thanks.

**Comment by kinsomicrote** [comment]
Been searching for how to get this to work. I think the cert needs to be updated.

**Comment by christopherstyles** [comment]
Ah, you’re right. I needed to update the cert before being able to release a new version. Try it now? I just released `v0.9.0`. You may need to re-add the cert as outlined in the [README#quick-start](https://github.com/christopherstyles/ngp_van#quick-start).

**Comment by kinsomicrote** [comment]
Got it.
Works fine now, thanks.
